rules/windows/impact_backup_file_deletion.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2021/10/01"
+maturity = "production"
+updated_date = "2021/10/01"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the deletion of Backup Files. Adversaries may delete Backup files
+to ensure that recovery from a Ransomware attack is less likely.
```

Comment:
maybe need to be linted (one line)

Reply:
I've added some more details, to the description